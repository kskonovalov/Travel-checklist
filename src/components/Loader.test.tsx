import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import Loader from './Loader';

configure({ adapter: new Adapter() });

const setUp = () => {
  return <Loader />;
};

describe('Component: Loader', () => {
  test('Should match snapshot', () => {
    const component = shallow(setUp());
    expect(component).toMatchSnapshot();
  });
});
