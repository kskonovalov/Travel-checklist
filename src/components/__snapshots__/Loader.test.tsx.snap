// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Loader Should match snapshot 1`] = `
<Fragment>
  <styled.div>
    <div />
    <div />
    <div />
  </styled.div>
</Fragment>
`;
